src/Paket.Core/Environment.fs

```diff
@@ -80,3 +80,14 @@ module PaketEnv =
     let ensureLockFileExists environment =
         environment.LockFile
         |> failIfNone (LockFileNotFound environment.RootDirectory)
+
+    let init (directory : DirectoryInfo) =
+        match locatePaketRootDirectory directory with
+        | Some rootDirectory -> 
+            Logging.tracefn "Paket is already initialized in %s" rootDirectory.FullName
+        | None -> 
+            let dependenciesFile = 
+                DependenciesFile(
+                    Path.Combine(directory.FullName, Constants.DependenciesFileName), 
+                    InstallOptions.Default, [], [], [])
+            dependenciesFile.Save()
```

Comment:
Actually this `Save` call can fail. For example insufficient rights to create a file.